### PR TITLE
Updated the About us section and rearranged consultants alphabetically

### PR DESCRIPTION
**Pull request by kememil**

### CODE REVIEW

File: content/about/_index.md [hunk 1]
```diff
@@ -5,27 +5,10 @@ menu: main
...
-that our solutions will meet their needs and beyond.
+Expert Analytics AS is a science and technology consultancy company located in Oslo, Norway. We offer high-quality technology solutions to both domestic and international customers. Our primary fields of expertise are data science, mathematical modelling and advanced back-end software development. The company has close ties to local research institutes and 2/3 of our employees hold a PhD in mathematics, physics or computer science. We have special expertise in high performance computing, advanced data visualization, data science and machine learning for extracting information from large unstructured data sets.
 
-We offer services within the following fields:
+Expert Analytics has a strong focus on research and development and has a keen interest in the use of computational methods and data science in the industry. The company therefore sees great value and mutual benefit in close interactions with both the scientific community and our industrial partners. Our main focus areas in the industry are towards the energy sector like wind and water power plants, heavy machinery and automation of manufacturing.
```
Comment (skavhaug) [inline content/about/_index.md:10]: hydro power plants

**Review by skavhaug — CHANGES_REQUESTED** [review]
Generally, I like this, but there are a couple of things to change. One simple, one more difficult :)

File: content/about/_index.md (hunk 1 above)
Comment (kememil) [reply content/about/_index.md:10]: Ok. I can change that to "wind- and hydropower plants".

File: content/about/_index.md [hunk 2]
```diff
@@ -5,27 +5,10 @@ menu: main
...
 
-- Data analysis
-- Numerical programming
-- Machine learning
-- General software development, such as applications, web, libraries,
-  integrations, scripting, etc.
-- Advisory services within ICT
-- Training
+Our experts have years of hands-on experience in computational science and modern industrial software development. This translates into an ability to analyze and solve complex challenges for the client, and it guarantees that our solutions will meet their needs and beyond.
```
[the remark this answers is not in the document]
Comment (kememil) [reply content/about/_index.md:12]: This is basically the same sentence as in the old About Us. And I agree that it is too fluffy. Let me come up with an alternative :)

[the remark this answers is not in the document]
Comment (kememil) [reply content/about/_index.md:12]: Suggestion:
Our experts have years of hands-on experience in computational science and robust industrial software development. We thrive on complex challenges and use scientific methods and technical excellence to develop flexible and tailored solutions that close the gap between needs and tools.

[the remark this answers is not in the document]
Comment (kememil) [reply content/about/_index.md]: @skavhaug Is this good for now? I think at least it is better than what is online right now, and then we can always refine :)

**Review by mandus — APPROVED** [review]
looks fine to me; about time to roll this out?

**Comment by grytveten** [comment]
@mandus, kan vi lukke denne eller er det fortsatt snakk om å merge?

**Comment by mandus** [comment]
> @mandus, kan vi lukke denne eller er det fortsatt snakk om å merge?

Det var vel et forsøk på å forbedre teksten - og det behovet er der fortsatt! Så i påvente av at vi får gjort en skikkelig oppgradering av websidene kan man kanskje ta en liten iterasjon på denne PRen og så pushe den ut?

**Comment by skavhaug** [comment]
I'll change the text to the one suggested by Emil, and then I think we should roll it out :)

**Comment by skavhaug** [comment]
Haha, 'twas changed already.

**Review by skavhaug — APPROVED** [review]

**Pull request merged**